src/routes/Sponsors.js

```diff
@@ -2,17 +2,37 @@
 // Email: khangpham@csu.fullerton.edu
 
 import NavBar from '../components/Navbar';
+import '../css/sponsors.css'
 
-
+function Text({name}){
+	return(
+		<div>
+		<p className="text">{name}</p>
+		</div>
+	)
+}
 function Sponsors() {
 	return(
 		<div>
 			<NavBar />
-			<p>Sponsors page goes here</p>
+			<body>
+			<h1><center><Text name={"Sponsors"} /></center></h1>
+			
+			<p>
+				<center>
```

Comment:
The text is centered but the centering is kind of awkward. I would put all of the information in one center tag. Also, add line spacing and increase the left and right margins to the text in the center tags to make it look nicer. 
Changing line height reference: https://developer.mozilla.org/en-US/docs/Web/CSS/line-height
Changing margin reference: https://developer.mozilla.org/en-US/docs/Web/CSS/margin

Reply:
I changed the line spacing and margins.